src/data_model/eng2math.jl

```diff
@@ -457,6 +456,7 @@ function _map_eng2math_transformer!(data_math::Dict{String,<:Any}, data_eng::Dic
                     "polarity"      => eng_obj["polarity"][w],
                     "tm_set"        => eng_obj["tm_set"][w],
                     "tm_fix"        => eng_obj["tm_fix"][w],
+                    "sm_ub"         => get(eng_obj, "sm_ub", Inf),
```

Comment:
I intended it to be `Real`. I discussed this a while back with @frederikgeth , and as a philosophy for bound specs we tried to stick as close as possible to how manufacturers supply ratings; for lines this means a per-conductor current rating, but for transformers it is a total apparent power rating. I kept on supporting the ones common in the OPF community as well (thermal rating for lines), but since these transformers bounds don't have strong convention, I went with the one that we believe makes most sense.